README.md

```diff
@@ -74,8 +74,24 @@ them together into one.
 Maintainers
 -------------------
 
-The project is maintained by
- - Nicco Kunzmann ([@niccokunzmann](https://github.com/niccokunzmann))
- - Tarun Kumar ([@meets2tarun](https://github.com/meets2tarun))
- 
- [gitter]: https://gitter.im/fossasia/open-event-orga-server
+You can reach the maintainers,
+ping them personally by loooking at the [Badgeyay team][team].
+
+You can become a maintainer by following the project and contributing code to it.
+Please see your role in the [CCCC][cccc].
+
+The project is maintained by the [Badgeyay maintainer's team][team].
+To join the team:
+1. Contribute
+2. You or someone else proposes you in an issue to become a member of the team.
+3. A Badgeyay admin adds you.
+
+To be excluded from the team:
+1. Do nothing or violate the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) willingly in a harmful way. 
+2. Someone creates an issue to ask for removal.
+3. An admin removes you.
+
+
+[gitter]: https://gitter.im/fossasia/open-event-orga-server
```

Comment:
@niccokunzmann change it to https://gitter.im/fossasia/badgeyay and then its good for merge

Reply:
We do not have a chat room for badgeyay yet! We are currently using open event room in gitter.

Reply:
@jajodiaraghav yes we have chat room. Check out this link below

 https://gitter.im/fossasia/badgeyay

Reply:
Sorry my bad!